cmatrix.c

```diff
@@ -243,14 +243,14 @@ void resize_screen(void) {
         return;
     }
 
-    COLS = win.ws_col;
-    LINES = win.ws_row;
+    COLS == win.ws_col;
+    LINES == win.ws_row;
```

Comment:
I don't understand. Why `==`?

Reply:
because of: 
`
gcc -DHAVE_CONFIG_H -I.     -g -O2 -MT cmatrix.o -MD -MP -MF .deps/cmatrix.Tpo -c -o cmatrix.o cmatrix.c
cmatrix.c: In function ‘resize_screen’:
cmatrix.c:277:10: error: lvalue required as left operand of assignment
     COLS = win.ws_col;
          ^
cmatrix.c:278:11: error: lvalue required as left operand of assignment
     LINES = win.ws_row;
           ^
cmatrix.c:282:15: error: lvalue required as left operand of assignment
         LINES = 10;
               ^
cmatrix.c:285:14: error: lvalue required as left operand of assignment
         COLS = 10;
              ^
make[1]: *** [Makefile:436: cmatrix.o] Fehler 1
`
https://www.thecrazyprogrammer.com/2017/08/lvalue-required-as-left-operand-of-assignment.html

Reply:
I know that. I mean `==` compares the value, but does not change it.

Reply:
??? No i dont think you understand that. Sorry. All is said.